lib/compat/wordpress-6.4/class-gutenberg-rest-templates-controller-6-4.php

```diff
@@ -72,4 +72,54 @@ protected function prepare_revision_links( $template ) {
 
 		return $links;
 	}
+
```

Comment:
What are these changes about. Also we can't make edits to files in old compat folders. If we want to make updates to this controller, we'd have to create a new class on top of this one in the `wordpress-6.6` folder.